src/screens/Customers/EditCustomer.tsx

```diff
@@ -28,11 +28,11 @@ function EditCustomer(props: EditCustomerProps) {
   return (
     <BaseScreen title="Edit Customer" leftIcon="backNav">
       <form noValidate className={classes.content}>
-        <TextField label={'Name'} />
+        <TextField label={'Name'} id={'name'}/>
```

Comment:
FYI I didn't work on this screen -- I just had to add `id`s when I merged in `main` since I made it a required prop. If you were wondering, we need `id` for accessibility!

Reply:
Yeah i think there was some merge weirdness with one of Tiffany's PRs a while back, thanks for putting it back!